src/util/github-utils.ts

```diff
@@ -1,12 +1,25 @@
 import { GitHubUserClient } from "~/src/github/client/github-user-client";
+import { GitHubInstallationClient } from "~/src/github/client/github-installation-client";
 import Logger from "bunyan";
+import { booleanFlag, BooleanFlags } from "config/feature-flags";
 
-export const isUserAdminOfOrganization = async (githubClient: GitHubUserClient, orgName: string, username: string, orgType: string, logger: Logger): Promise<boolean> => {
+export const isUserAdminOfOrganization = async (githubClient: GitHubUserClient, jiraHost: string, gitHubInstallationClient: GitHubInstallationClient, orgName: string, username: string, orgType: string, logger: Logger): Promise<boolean> => {
 	if (orgType === "User") {
 		logger.info("isUserAdminOfOrganization: orgType is a user");
 		return orgName === username;
 	}
 
+	try {
+		if (await booleanFlag(BooleanFlags.USE_INSTALLATION_CLIENT_CHECK_PERMISSION, jiraHost)) {
+			logger.info("isUserAdminOfOrganization: orgType is an org, checking membership (app client)");
+			const { data: { state, role } } = await gitHubInstallationClient.getUserMembershipForOrg(username, orgName);
+			logger.info({ orgName, username, state, role }, `isUserAdminOfOrganization: User has role for org`);
+			return role === "admin" && state === "active";
```

Comment:
curious why do we need the second check? it wasn't in the version with the user client...

Reply:
I don't know. Maybe we didn't do it right at the first place? Like inactive admin can still do sh**?